documentation/src/main/asciidoc/topics/config_examples/server_ldap_realm.xml

```diff
@@ -16,7 +16,8 @@
                     direct-verification="true">
             <!-- Defines how principals are mapped to LDAP entries. -->
             <identity-mapping rdn-identifier="uid"
-                              search-dn="ou=People,dc=infinispan,dc=org">
+                              search-dn="ou=People,dc=infinispan,dc=org"
+                              search-recursive="false">
```

Comment:
Should we add documentation for the filter and iterator filter? I, personally, have no clue what that is :)

Reply:
I've added a comment about `filter-name` but removed `iterator-filter` completely as it makes no sense for our use-case (it is used when retrieving multiple users at once).